### ISSUE TITLE
Message output not visible on linux when the background is black

### ISSUE THREAD

**Issue by moozzyk** [Microsoft/msbuild#754]
```
<?xml version="1.0"?>
<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <Target Name="Build">
        <Message Text="********************ABVC" />
    </Target>
</Project>
```

When the background is black executing the Build target gives you: 
![image](https://cloud.githubusercontent.com/assets/1438884/16534453/baae1250-3f94-11e6-8058-1ba56f22a1c0.png)

Change the background color and messages magically appear: 

![image](https://cloud.githubusercontent.com/assets/1438884/16534466/ca9be818-3f94-11e6-9e3f-5ce2990e4518.png)

Environment:

```
NAME="Ubuntu"
VERSION="14.04.4 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.4 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
```

**Comment by moozzyk** [comment]
This can be workaround by using `/consoleloggerparameters:DisableConsoleColor` but it is not a long term solution. (credit goes to @anurse)

**Comment by anurse** [comment]
For clarity, the first message in that example ("MSBuild already initialized...") is actually from our (ASP.NET) own build scripts, so I'm fixing that one ;)

**Comment by Sarabeth-Jaffe-Microsoft** [comment]
We aren't familiar with KoreBuild. Can you reproduce this issue running MSBuild directly?

- MSBuild team

**Comment by anurse** [comment]
KoreBuild is just our (ASP.NET) scripts to download and launch MSBuild Core. The below messages come directly from MSBuild:

![image](https://cloud.githubusercontent.com/assets/7574/16786487/ee887e86-4849-11e6-8a7a-18edf30920f4.png)

**Comment by colemickens** [comment]
Yup. This happens with raw `dotnet build` in the `docker.io/microsoft/dotnet:1.1.0-sdk-msbuild` image.

<img width="1507" alt="screen shot 2016-12-15 at 3 08 15 pm" src="https://cloud.githubusercontent.com/assets/327028/21245603/5a199d2a-c2d8-11e6-9c45-f60d95940e3f.png">

**Comment by livarcocc** [comment]
Can someone provide a repro for this issue? I can't seem to reproduce it normally.
